test/unit/tasks.spec.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
```

Comment:
I think we should remove **.spec** file without tests. Unless that file is something we need and I don't know about it?